erizo/src/test/utils/Matchers.h

```diff
@@ -0,0 +1,24 @@
+#ifndef ERIZO_SRC_TEST_UTILS_MATCHERS_H_
+#define ERIZO_SRC_TEST_UTILS_MATCHERS_H_
+
+#include <rtp/RtpHeaders.h>
+#include <MediaDefinitions.h>
+
```

Comment:
Could we have this matchers also under the `erizo` namespace so it's easier to identify that they are part of the test utils?